tests/multiparser_js_css/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -264,17 +265,18 @@ styled.div\`
 \`;
 
 styled.span\`
-  \${foo}
-  \${bar}
+  \${foo};
```

Comment:
Anyone know is this going tobe a problem?

Reply:
https://github.com/prettier/prettier/issues/2350
https://github.com/prettier/prettier/issues/2636

Reply:
if you're sure about this I'll try to remove this extra semi, it can be done

Reply:
Yes, that would be great!

Reply:
done.